.github/workflows/ci.yml

```diff
@@ -22,6 +22,7 @@ jobs:
         uses: golangci/golangci-lint-action@v3
         with:
           version: latest
+          skip-cache: true
```

Comment:
Hm, the reason this is okay is because the `cache: true` on line 19?

Reading the code comments of an action yaml file at https://github.com/golangci/golangci-lint-action#how-to-use surfaces nothing except "if you set this to true, there will be no caching whatsoever"

Does this result in a faster runtime?

Another Q is about this part of the readme:

> We recommend running this action in a job separate from other jobs (go test, etc) because different jobs [run in parallel](https://help.github.com/en/actions/getting-started-with-github-actions/core-concepts-for-github-actions#job).

Would it make sense to put this into its own yaml file? I _think_ that's what they are referring to when saying "running this action separate from others", and I think currently `lint` and `test` are under the same `jobs` parent.

Otherwise I'm giving it the go-ahead.

Reply:
> Hm, the reason this is okay is because the cache: true on line 19?

Correct, I'm trying to use the cache that other jobs would use for go mods. This way the golangci-lint action doesn't need to download them again every time.

> Does this result in a faster runtime?

I was hoping so, but it didn't use the cache from the step above and [reinstalled everything](https://github.com/suborbital/e2core/actions/runs/3528085171/jobs/5917805441):
```
Ran golangci-lint in 65830ms
```

> Would it make sense to put this into its own yaml file?

I believe they mean run lint separately to a test as a separate job. A different file would be a different workflow. Also, because the golangci-lint action can handle its own caching it's kinda magic in that way to run independently and quickly.

I don't like how they use their own cache because it's ~500mb per build and ... we already store it once as gomods 😅 See the duplication? 

<img width="742" alt="Screenshot 2022-11-23 at 5 24 12 PM" src="https://user-images.githubusercontent.com/7211830/203656077-63983fa5-0e41-4b52-9f10-8f32e58f5ad6.png">

Unfortunately this PR doesn't fix that problem yet 😭

Reply:
Right. Hm... All right, :shipit: !